rosdep/python.yaml

```diff
@@ -3197,3 +3197,9 @@ xdot:
   debian: [xdot]
   fedora: [python-xdot]
   ubuntu: [xdot]
+python-argh:
+  ubuntu:
```

Comment:
It looks like python-argh is generally available on all versions. Could you change this rule to not call out specific versions: http://packages.ubuntu.com/search?keywords=python-argh